.circleci/config.yml

```diff
@@ -54,6 +54,31 @@ jobs:
           # Publish *.jar
           gradle publish
 
+  build-integration-spark:
+    working_directory: ~/openlineage/integration/spark
+    docker:
+      - image: circleci/openjdk:8-jdk
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-integration-spark-{{ .Branch }}-{{ .Revision }}
+            - v1-integration-spark-{{ .Branch }}
+      - run: ./integration/spark/gradlew -p integration/spark/ --no-daemon --stacktrace build
```

Comment:
Do you need to prefix the gradle cmds with `./integration/spark` if the CI working directly is `~/openlineage/integration/spark`? All cmds should be relative to the working dir.

Reply:
As I got working dir, is where the code should be checkout, it's my view from my tests. I'll double check that

Reply:
I've changed  -checkout step to - *checkout_project_root, and ./gradlew works fine without prefixes